config/tsconfig.base.json

```
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
```

```suggestion
  "compilerOptions": {
    "module": "nodenext",
```

https://www.typescriptlang.org/tsconfig#module
> You very likely want "nodenext" for modern node projects.

---

nvm looks like node16 anyway (and should do this for the `moduleResolution` too)